const.py

```diff
@@ -1,10 +1,14 @@
-class AccountType(object):
-    NORMAL = 0
-    IRONMAN = 1
-    HARDCORE_IRONMAN = 2
-    ULTIMATE_IRONMAN = 3
-    DEADMAN = 4
-    DEADMAN_SEASONAL = 5
+from enum import Enum, unique
+
+
+@unique
+class AccountType(Enum):
+    NORMAL           = "/m=hiscore_oldschool/index_lite.ws"
+    IRONMAN          = "/m=hiscore_oldschool_ironman/index_lite.ws"
+    HARDCORE_IRONMAN = "/m=hiscore_oldschool_hardcore_ironman/index_lite.ws"
+    ULTIMATE_IRONMAN = "/m=hiscore_oldschool_ultimate/index_lite.ws"
+    DEADMAN          = "/m=hiscore_oldschool_deadman/index_lite.ws"
+    SEASONAL         = "/m=hiscore_oldschool_seasonal/index_lite.ws"
```

Comment:
It looks like Twisted League is under the"DEADMAN_SEASONAL" link, so I changed this enum to "SEASONAL".